src/test/java/uk/gov/dhsc/htbhf/BrowserStackConfiguration.java

```diff
@@ -24,7 +24,7 @@
     @Value("${wait.timeout.seconds}")
     private int waitTimeoutInSeconds;
 
-    @Bean(destroyMethod = "closeDriver")
+    @Bean
```

Comment:
We do not need this for BrowserStack tests as we're calling `quit` at the end of each test in `Hooks`.